Packs/AzureActiveDirectory/ReleaseNotes/1_2_11.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Azure Active Directory Identity Protection  (Deprecated)
+- Improved implementation of empty response handling in MicrosoftApiModule.
```

Comment:
`MicrosoftApiModule` makes little sense to the average user.
Please make sure to add it to all relevant files.
```suggestion
- Improved handling of empty responses.
```